ProfileManager.js

```diff
@@ -355,6 +371,29 @@ export default class ProfileManager {
       }));
     }
 
+    const profileAgentDetails = await this.getProfileAgent({profileAgent});
+    const referenceId = edvs.getReferenceId('users-edv-document');
```

Comment:
@gannan08 I'm in favor of making the code more clear, but I think your suggestion has the opposite effect, and additionally makes code text search for `users-edv-document` impossible.

How about this?
```
const documentReferenceId = edvs.getReferenceId('users-edv-document');
```

Also, we might consider renaming this helper to `createReferenceId`.

Reply:
I think your change would be a step in the right direction. The original intention was to create reference ids just for EDVs with a specific format to prevent collisions. I'm open to accepting your change and discuss any further refactoring offline.

Reply:
Looking at some other usage of `getReferenceId`.  Maybe we need to consider a more sophisticated naming convention in these referenceIds.  The existing convention appears to be:
```
<host>:<edv name>-[<some>-<thing>]
```
without some other sentinel character (not `-`) between the EDV and `some-thing`, it's impossible to distinguish the EDV name from the `some-thing`.